assets/cubyz/shaders/skybox/sky.fs

```diff
@@ -0,0 +1,11 @@
+#version 430
+
+layout (location=0) out vec4 fragColor;
+
+in vec3 pos;
+
+uniform vec3 skyColor;
+
+void main() {
+	fragColor = vec4(skyColor, 1);
+}
```

Comment:
Do we even need this shader anymore? It seems to just do what `glClear` is already doing, setting everything to one color, except it costs an additional 1.2 ms per frame for me.